docs/connectors/sources/http.md

```diff
@@ -1,16 +1,87 @@
-# HTTP Input
+# HTTP input connector
 
-Feldera supports pushing data to a SQL table over HTTP.  For more information,
-see
+Feldera supports directly pushing data to a SQL table over HTTP.
 
-* [Tutorial section](/docs/tutorials/basics/part2) on HTTP-based input and
-  output.
+* This input connector is ephemeral: it is created when the HTTP
+  connection is opened, and is deleted once it is closed.
 
-* [REST API documentation](https://www.feldera.com/api/push-data-to-a-sql-table)
-  for the `/ingress` endpoint.
+* It is the only input connector not created and managed by the user.
```

Comment:
and for this reason I wouldn't document it as a connector. It's really just part of the API, connectors are an implementation detail in this case.

Reply:
I think it could be worthwhile to see it as a perpetual connector, and possibly implement it as such (see the other comment thread with @lalithsuresh )